pelican/utils.py

```diff
@@ -646,22 +638,37 @@ def process_translations(content_list, order_by=None):
                 for x in lang_items:
                     logger.warning('\t%s', x.source_path)
 
+        # items with `translation` metadata will be used as translations...
+        candidate_items = list(filter(
+            lambda i:
+                i.metadata.get('translation', 'false').lower() == 'false',
+            items))
+        # ...unless all items with that slug are translations
+        if not candidate_items:
+            logger.warning('All times of "%s" are translations', slug)
+            candidate_items = items
+
         # find items with default language
-        default_lang_items = list(filter(
+        original_items = list(filter(
             attrgetter('in_default_lang'),
-            default_lang_items))
+            candidate_items))
+
+        # if there is no article with default language, go back one step
+        if not original_items:
+            original_items = candidate_items
 
-        # if there is no article with default language, take an other one
-        if not default_lang_items:
-            default_lang_items = items[:1]
+        # display warning if there are several base items
+        if len(original_items) > 1:
+            logger.warning('There are %s original (not translated) items "%s"',
+                           len(original_items), slug)
 
         if not slug:
             logger.warning(
```

Comment:
Newline in log messages is perfectly fine since a while back (#1412) I added a decent way to display them :). However it only handles newlines in raw, unformatted messages (first argument). I might look into that to format the final message instead so it'll be possible to use newlines in the arguments as well.

But for now, you can do something like this:

```python
if not slug:
    logger.warning(
        'There are %s items with empty slug' + '\n%s'*len(items),
        len(items),
        *(x.source_path for x in items))
```

Which should output something like:

```
WARNING: There are 2 items with empty slug
  | /tmp/foo/a.md
  | /tmp/foo/b.md
```

Similarly for others.

As for the changes, a separate commit in this PR is fine I think.